server/src/routes/index.js

```diff
@@ -0,0 +1,16 @@
+
+import restRoutes from './rest';
+
+const init = (server) => {
+  server.get('*', (req, res, next) => {
+    console.log(`Request received for ${req.originalUrl}`);
+    return next();
```

Comment:
More of a learning question. What does return next() do here? :slightly_smiling_face:

Reply:
@karansapolia read about middlewares in express.
but if I had to give you a gist I would say that in express middlewares are functions which are run through every request unless you return a response.
let's say you have a request coming and you want to pass it throught three functions (middlewares) each adding something to the request or modifying it, since you are not return anything from those  middlewares, it will never go out of first middleware so for this you use next(). After the end of every middlware you call next() this actually calls the next middlware unlessa middleware or function sends a response/ throws an error or returns something

Reply:
This is a part of middleware.
You can read more about this [here](https://expressjs.com/en/guide/using-middleware.html).